intro.js

```diff
@@ -815,6 +834,11 @@
    * @param {Object} targetElement
    */
   function _showElement(targetElement) {
+    // try to requery element if it is dynamic
+    if(!document.body.contains(targetElement.element)) {
+      targetElement.element = document.querySelector(targetElement.selector);
```

Comment:
@HuggableSquare here `selector` is used.

Reply:
Ahh I see, missed that. What's going on with that? I was pretty sure I added support for elements that were inserted into the DOM later on here: https://github.com/HuggableSquare/intro.js/commit/004f82d260312a4e31af673daba8a7ba5daed134

Reply:
@HuggableSquare your change does not count elements that were removed from DOM after steps initialization.